### ISSUE TITLE
@def does not support package contained in the return type name 

### ISSUE THREAD

**Issue by greenlaw110** [greenlaw110/Rythm#249]
The following code will trigger compilation error:

```
@def org.rythmengine.RythmEngine rythm() {
    return __engine();
}
```

**Comment by WolfgangFahl** [comment]
had the same issue today - would be great if it gets fixed

**Comment by greenlaw110** [comment]
Work around:

```
@import org.rythmengine.RythmEngine

...

@def RythmEngine rythm() {
   return __engine();
}
```

**Comment by greenlaw110** [comment]
Fixed in 1.1.1-SNAPSHOT
